src/datepicker/datepicker.spec.ts

```diff
@@ -259,20 +259,31 @@ describe('ngb-datepicker', () => {
     expect(fixture.debugElement.query(By.directive(NgbDatepickerNavigation))).toBeNull();
   });
 
-  it('should override outside days to "hidden" if there are multiple months displayed', () => {
-    const fixture = createTestComponent(
-        `<ngb-datepicker [displayMonths]="displayMonths" [outsideDays]="'collapsed'"></ngb-datepicker>`);
-
+  it('should set outside days to "visible" if not provided when there is a single month displayed', () => {
+    const fixture = createTestComponent(`<ngb-datepicker [displayMonths]="1"></ngb-datepicker>`);
+    let months = fixture.debugElement.queryAll(By.directive(NgbDatepickerMonthView));
+    expect(months[0].componentInstance.outsideDays).toBe('visible');
+  });
 
+  it('should set outside days to "hidden" if provided when there is a single month displayed', () => {
+    const fixture = createTestComponent(`<ngb-datepicker [displayMonths]="1" outsideDays="hidden"></ngb-datepicker>`);
     let months = fixture.debugElement.queryAll(By.directive(NgbDatepickerMonthView));
-    expect(months[0].componentInstance.outsideDays).toBe('collapsed');
+    expect(months[0].componentInstance.outsideDays).toBe('hidden');
+  });
 
-    fixture.componentInstance.displayMonths = 2;
-    fixture.detectChanges();
-    months = fixture.debugElement.queryAll(By.directive(NgbDatepickerMonthView));
+  it('should set outside days to "hidden" if not provided when there are multiple months displayed', () => {
+    const fixture = createTestComponent(`<ngb-datepicker [displayMonths]="2"></ngb-datepicker>`);
+    let months = fixture.debugElement.queryAll(By.directive(NgbDatepickerMonthView));
     expect(months[0].componentInstance.outsideDays).toBe('hidden');
```

Comment:
Just a minor nitpick if you want to make the adjustment, you could do
```ts
let [month] = fixture.debugElement.queryAll(By.directive(NgbDatepickerMonthView));
expect(month.componentInstance.outsideDays).toBe('hidden');
```
or
```ts
let [{componentInstance: {outsideDays}}] = fixture.debugElement.queryAll(By.directive(NgbDatepickerMonthView));
expect(outsideDays).toBe('hidden');
```
Whatever is preferred is I guess whatever makes it more readable, just food for thought.